frontend/src/bundles/common/enums/app-route.enum.ts

```diff
@@ -1,7 +1,13 @@
 const AppRoute = {
     ROOT: '/',
+    LOGOUT: '/logout',
+    HELP: '/help',
     SIGN_IN: '/sign-in',
     SIGN_UP: '/sign-up',
+    OVERVIEW: '/overview',
+    GOALS: '/goals',
+    WORKOUT: '/workout',
+    SCHEDULE: '/schedule',
```

Comment:
I can not find these pages. 
Mb it better to mock it
`<div>SCHEDULE PAGE</div>`

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/556d67cf-7898-4a69-9858-99f1278095fa)